i18n/strings.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/* THIS IS A GENERATED FILE. DO NOT EDIT DIRECTLY */
+$i18nStrings = array(
+"Saving…" => __( "Saving…" ),
```

Comment:
A tab would be nice for everything inside `array()`
Also, any reason for using double quotes? We generally try to use single quotes by default, and double quotes when there is an advantage.